larpandoracontent/LArHelpers/LArPfoHelper.h

```diff
@@ -387,6 +387,14 @@ class LArPfoHelper
      */
     static bool SortByNHits(const pandora::ParticleFlowObject *const pLhs, const pandora::ParticleFlowObject *const pRhs);
 
+    /**
+     *  @brief  Retrieve a linearised representation of the PFO hierarchy in breadth first order
+     *
```

Comment:
As per MCParticleHelper (although less urgency, as role of existing individual functions is clearer in PfoHelper)